docs/content/docs.md

```diff
@@ -279,9 +280,9 @@ However using the connection string syntax, this does work.
 
     rclone copy "gdrive,shared_with_me:shared-file.txt" gdrive:
 
-Note that the connection string only affects the options of the immediate 
-backend. If for example gdriveCrypt is a crypt based on gdrive, then the 
-following command **will not work** as intended, because 
+Note that the connection string only affects the options of the immediate
```

Comment:
Can you drop the whitespace changes from this file please - it makes it very difficult to see what you changed - thanks